aptos-move/framework/aptos-framework/sources/vesting.spec.move

```diff
@@ -99,49 +139,102 @@ spec aptos_framework::vesting {
     spec create_vesting_contract {
         // TODO: Data invariant does not hold.
         pragma verify = false;
+        aborts_if withdrawal_address == @aptos_framework || withdrawal_address == @vm_reserved;
+        aborts_if !exists<account::Account>(withdrawal_address);
+        aborts_if !exists<coin::CoinStore<AptosCoin>>(withdrawal_address);
+        aborts_if len(shareholders) == 0;
+        aborts_if simple_map::spec_len(buy_ins) != len(shareholders);
     }
 
     spec unlock_rewards(contract_address: address) {
         // TODO: Calls `unlock_stake` which is not verified.
+        // Current verification times out.
         pragma verify = false;
+        include UnlockRewardsAbortsIf;
+    }
+
+    spec schema UnlockRewardsAbortsIf {
+        contract_address: address;
+
+        // Cause timeout here
```

Comment:
Is this the same issue as the one above? Is it proved with a enough time-limit?

Reply:
yes, its the same issue